iceoryx_hoofs/design/include/iox/newtype.hpp

```diff
@@ -121,6 +124,12 @@ class NewType : public Policies<NewType<T, Policies...>>...
     template <typename Type>
     friend typename Type::value_type newtype::internal::newTypeAccessor(const Type&) noexcept;
 
+    template <typename Type>
+    friend typename Type::value_type newtype::internal::preIncrement(Type&) noexcept;
+
+    template <typename Type>
+    friend typename Type::value_type newtype::internal::preDecrement(Type&) noexcept;
```

Comment:
Having a `Type::value_type& newtype::internal::newTypeAccessor(Type&)` instead these two functions would ease the implementation of `operator+=`

Reply:
:heavy_check_mark: